### PR TITLE
admin styles – result list and filter, toolbar

### PR DESCRIPTION
**Pull request by lory87**
- working on admin style changes
- added a new sass file for changelist and removed the styles from content sass file
- please ignore djangocms_admin_style/static/djangocms_admin_style/css/djangocms-admin.css

updates:
- result list and filter
- toolbar
- search button
- fixes

### CODE REVIEW

File: djangocms_admin_style/sass/partials/_content.scss [hunk 1]
```diff
@@ -293,16 +295,23 @@ body.login {
...
+    top: 0;
+    right: 45px;
+    left: 45px;
+    color: $font-color-light !important;
+    font-size: 15px;
+    margin-top: 65px;
+    padding: 20px 10px;
+    border-top: 0;
+    border-bottom: 1px solid $line-color-light;
+    background: none !important;
```
Comment (Brankub) [inline djangocms_admin_style/sass/partials/_content.scss]: important :(

File: djangocms_admin_style/sass/partials/_content.scss [hunk 2]
```diff
@@ -293,16 +295,23 @@ body.login {
...
-    padding-bottom: 4px;
-    padding-left: 25px;
-    border-top: 1px solid $line-color-dark;
-    border-bottom: 0;
-    background: $background-color-dark;
+    position: absolute;
+    top: 0;
+    right: 45px;
+    left: 45px;
+    color: $font-color-light !important;
```
Comment (Brankub) [inline djangocms_admin_style/sass/partials/_content.scss]: important :(

File: djangocms_admin_style/sass/partials/_content.scss [hunk 3]
```diff
@@ -14,14 +14,15 @@
...
-    width: auto !important;
+#sitemap {
+    clear: both;
 }
 
 // module
 .module {
     @include box();
     margin-bottom: 10px;
+    background: none !important;
```
Comment (Brankub) [inline djangocms_admin_style/sass/partials/_content.scss]: important :(

File: djangocms_admin_style/sass/partials/_content.scss (hunk 1 above)
Comment (lory87) [inline djangocms_admin_style/sass/partials/_content.scss]: yes have to overwrite it. not possible to not use it without important

**Comment by lory87** [comment]
@Brankub have to overwrite these things. sorry not possible to remove !important

**Comment by vxsx** [comment]
screenshots of changes would be cool actually, but that shouldn't block you
looks good to me

**Comment by Brankub** [comment]
do not like import, but LGTM

**Pull request merged**